affiliates/colombia.swagger.json

```diff
@@ -0,0 +1,486 @@
+          }
+        }
+      },
+      "ValidationError" : {
+        "description" : "Validation error",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/ValidationErrors"
+            }
+          },
+          "application/xml" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/ValidationErrors"
+            }
+          }
+        }
+      },
+      "Unauthorized" : {
+        "description" : "Unauthorized",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/UnauthorizedResponse"
+            }
+          },
+          "application/xml" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/UnauthorizedResponse"
+            }
+          }
+        }
+      }
+    },
+    "schemas" : {
+      "PrimaryAddress" : {
+        "type" : "object",
+        "required" : [ "city", "street"],
+        "properties" : {
+          "postal_code" : {
+            "type" : "string",
+            "description" : "Format: /^[0-9]{6}$/",
+            "example" : "989344"
+          },
+          "city" : {
+            "type" : "string",
+            "example" : "San Santiagodel Penedès",
+            "description" : "Format: /^[a-záéèíñóúüç[:space:]\\-\\']+$/ui"
+          },
+          "street" : {
+            "type" : "string",
+            "example" : "Calle Díaz"
+          },
+          "house_number" : {
+            "type" : "string",
+            "example" : "11"
+          },
+          "flat_number" : {
+            "type" : "string",
+            "example" : "4"
+          }
+        }
+      },
+      "ContactAddress" : {
+        "type" : "object",
+        "required" : [ "postal_code", "city", "street", "house_number"],
+        "properties" : {
+          "postal_code" : {
+            "type" : "string",
+            "description" : "Format: /^[0-9]{6}$/",
+            "example" : "989344"
+          },
+          "city" : {
+            "type" : "string",
+            "example" : "San Santiagodel Penedès",
+            "description" : "Format: /^[a-záéèíñóúüç[:space:]\\-\\']+$/ui"
+          },
+          "street" : {
+            "type" : "string",
+            "example" : "Calle Díaz"
+          },
+          "house_number" : {
+            "type" : "string",
+            "example" : "11"
+          },
+          "flat_number" : {
+            "type" : "string",
+            "example" : "4"
+          }
+        }
+      },
+      "Agreements" : {
+        "type" : "object",
+        "required" : [ "data_proccessing_policy", "terms_of_service" ],
+        "properties" : {
+          "data_proccessing_policy" : {
+            "type" : "boolean",
+            "example" : true
+          },
+          "terms_of_service" : {
+            "type" : "boolean",
+            "example" : true
+          }
+        }
+      },
+      "Customer" : {
```

Comment:
Can you list only the fields that we have on https://www.credy.com.co/registro-prestamo-corto/#contact ? For example exmployer or employer_phone is not even asked on the form, so not worth a mention.

Reply:
removed unnecessary fields